From d7484cd98051aa1588683136da0bb99eac31523b Mon Sep 17 00:00:00 2001
From: Martin Weinelt <hexa@darmstadt.ccc.de>
Date: Wed, 7 Feb 2024 15:03:48 +0100
Subject: [PATCH] Replace remaining usage of assertEquals with assertEqual

This fixes the tests on Python 3.12, where assertEquals was removed.
---
 Lib/fontParts/test/test_glyph.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/Lib/fontParts/test/test_glyph.py b/Lib/fontParts/test/test_glyph.py
index 1cad7814..20d01ae6 100644
--- a/Lib/fontParts/test/test_glyph.py
+++ b/Lib/fontParts/test/test_glyph.py
@@ -1356,9 +1356,9 @@ def test_isEmpty_false_component(self):
 
     def test_removeOverlap(self):
         glyph = self.getGlyph_generic()
-        self.assertEquals(len(glyph), 2)
+        self.assertEqual(len(glyph), 2)
         glyph.removeOverlap()
-        self.assertEquals(len(glyph), 1)
+        self.assertEqual(len(glyph), 1)
 
 
 def test_generator(test_name, metric, value):
